### ISSUE TITLE
1.50.x Password files incompatible with 1.51.x

### ISSUE THREAD

**Issue by jason-oliveira** [brave/brave-browser#30040]
## Description 

Password lists and sync from 1.50.125 and earlier do not work on v1.51.106 (marked "release")

## Steps to Reproduce

   1. Install 1.50.125.
   2. Log into a few places and allow Brave to save the passwords.
   3. Install 1.51.106
   4. check your password list and see that they're gone.
   5. install 1.50.125 again.
   6. check your password list and see that the passwords have returned.

## Actual result:

Password list is completely empty, other than sites set to disable passwords.

## Expected result:
Password list is populated with passwords from previous version.

## Reproduces how often: 

Easily reproduced - Every gentoo user using my `brave-overlay` is currently experiencing the problem, so I had to mark 1.51.106 as unstable. :(

## Brave version (brave://version info)

1.51.106 is affected, but is probably not isolated; I have not tested previous (beta) releases of the 1.51.x branch.

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? no 
- Does the issue resolve itself when disabling Brave Rewards? no
- Is the issue reproducible on the latest version of Chrome? no

## Miscellaneous Information:

I have been able to work-around the issue by exporting passwords to csv in 1.50.125 and re-importing successfully in 1.51.106.

**Comment by Finoderi** [comment]
I'm nor sure the password incompatibility is the actual problem here. On gentoo Brave 1.51.* can't read anything from kde wallet for some reason. So the brave-sync doesn't work hence there are no passwords from previous version.
I don't know if it's another Chromium fuck up but it's getting really annoying.

**Comment by Finoderi** [comment]
Yeah, on 1.50.125 everything works fine. On 1.51.110 password for Brave storage in KDE wallet changes every time the browser is launched. So it can read anything from previously stored data. Unbelievable...
